Gemfile

```diff
@@ -1,2 +1,2 @@
 source 'https://rubygems.org'
-gem 'github-pages'
+gem 'github-pages', group: :jekyll_plugins
```

Comment:
This change allows the github-pages gem to set jekyll config settings, like enabling Github flavored markdown.  The full config it's using is [in this file](https://github.com/github/pages-gem/blob/master/lib/github-pages/configuration.rb).